README.md

```diff
@@ -43,6 +44,15 @@ sudo snap run opensearch.setup          \
     --tls-init-setup yes    # this creates the root and admin certs as well.
 ```
 
+#### (Optional) Set `snap_daemon` homedir:
```

Comment:
I am not entirely why we're doing this, opensearch doesn't need / require the home_dir, can you explain?

Reply:
We can. However, I am not sure how it will play out if we have 2x snaps that need `snap_daemon` and `homedir`

Reply:
@Mehdi-Bendriss the `home` plugin works by giving `snap_daemon` access to the calling user, not setting up `snap_daemon`'s home.

I tried setting `HOME=${SNAP_COMMON}/...` but did not work either. The only way to get it is really doing `usermod`, and the snap does not allow that.